code/modules/client/preferences_savefile.dm

```diff
@@ -143,87 +148,94 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 
 /datum/preferences/proc/load_preferences()
 	if(!path)
-		return 0
+		return FALSE
 	if(!fexists(path))
-		return 0
+		return FALSE
 
 	var/savefile/S = new /savefile(path)
 	if(!S)
-		return 0
+		return FALSE
 	S.cd = "/"
 
 	var/needs_update = savefile_needs_update(S)
 	if(needs_update == -2)		//fatal, can't load any data
-		return 0
+		return FALSE
 
 	//general preferences
-	S["asaycolor"]			>> asaycolor
-	S["ooccolor"]			>> ooccolor
-	S["lastchangelog"]		>> lastchangelog
-	S["UI_style"]			>> UI_style
-	S["overhead_chat"]		>> overhead_chat
-	S["outline_color"]		>> outline_color
-	S["outline_enabled"]	>> outline_enabled
-	S["hotkeys"]			>> hotkeys
-	S["tgui_fancy"]			>> tgui_fancy
-	S["tgui_lock"]			>> tgui_lock
-	S["buttons_locked"]		>> buttons_locked
-	S["windowflash"]		>> windowflashing
-	S["be_special"] 		>> be_special
-
-	S["crew_objectives"]	>> crew_objectives
-
-
-	S["default_slot"]		>> default_slot
-	S["chat_toggles"]		>> chat_toggles
-	S["toggles"]			>> toggles
-	S["ghost_form"]			>> ghost_form
-	S["ghost_orbit"]		>> ghost_orbit
-	S["ghost_accs"]			>> ghost_accs
-	S["ghost_others"]		>> ghost_others
-	S["preferred_map"]		>> preferred_map
-	S["ignoring"]			>> ignoring
-	S["ghost_hud"]			>> ghost_hud
-	S["inquisitive_ghost"]	>> inquisitive_ghost
-	S["uses_glasses_colour"]>> uses_glasses_colour
-	S["clientfps"]			>> clientfps
-	S["parallax"]			>> parallax
-	S["ambientocclusion"]	>> ambientocclusion
-	S["auto_fit_viewport"]	>> auto_fit_viewport
-	S["pixel_size"]	    	>> pixel_size
-	S["scaling_method"]	    >> scaling_method
-	S["menuoptions"]		>> menuoptions
-	S["enable_tips"]		>> enable_tips
-	S["tip_delay"]			>> tip_delay
-	S["pda_style"]			>> pda_style
-	S["pda_color"]			>> pda_color
-	S["show_credits"] >> show_credits
-
-	S["key_bindings"]		>> key_bindings
-
-	S["purchased_gear"]					>> purchased_gear
-	S["equipped_gear"]					>> equipped_gear
+	READ_FILE(S["asaycolor"], asaycolor)
+	READ_FILE(S["ooccolor"], ooccolor)
+	READ_FILE(S["lastchangelog"], lastchangelog)
+	READ_FILE(S["UI_style"], UI_style)
+	READ_FILE(S["overhead_chat"], overhead_chat)
+	READ_FILE(S["outline_color"], outline_color)
+	READ_FILE(S["outline_enabled"], outline_enabled)
```

Comment:
Heads up, you should entirely remove now-obsoleted `outline_enabled` pref from the codebase if you can.